contracts/Chaosnet.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity 0.8.9;
+
+/// @title Chaosnet
+/// @notice This is a beta staker program for stakers willing to go the extra
+/// mile with monitoring, share their logs with the dev team, and allow to more
+/// carefully monitor the bootstrapping network. As the network matures, the
+/// beta program will be ended.
+contract Chaosnet {
+  /// @notice Indicates if the chaosnet is active. The chaosnet is active
+  /// after the contract deployment and can be ended with a call to
+  /// `deactivateChaosnet()`. Once deactivated chaosnet can not be activated
+  /// again.
+  bool public isChaosnetActive;
+
+  /// @notice Indicates if the given operator is a beta operator for chaosnet.
+  mapping(address => bool) public isBetaOperator;
+
+  /// @notice Address controlling chaosnet status and beta operator addresses.
+  address public chaosnetMaestro;
```

Comment:
Not crucial and I like the "maestro" naming, but usually we use "manager" for the roles like this. Up to you.

Reply:
I would lean towards naming it `chaosnetManager` or `chaosnetOwner`.

Reply:
1ad78dfa8babd0980ac9ef891d19b2e360bd0699